### PR TITLE
CY-238: Windows privilege accesses tables added. AD dashboard seperat…

### PR DESCRIPTION
**Pull request by VatsalJagani**
…ed from windows dashboard.

### CODE REVIEW

File: cyences_app_for_splunk/default/data/ui/views/cs_windows_reports.xml [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      </input>
+      <input type="text" token="tkn_user" searchWhenChanged="true">
+        <label>User</label>
+        <default>*</default>
+        <prefix>*</prefix>
+        <suffix>*</suffix>
+      </input>
+    </panel>
+  </row>
+  <row depends="$hidden_row$">
```
Comment (babhalodia) [inline cyences_app_for_splunk/default/data/ui/views/cs_windows_reports.xml:56]: This seems like duplicated. We have same search at line no. 7

**Review by babhalodia — COMMENTED** [review]
Are we going to add any alerts for this?

**Comment by VatsalJagani** [comment]
Bro, as of now we are not gonna add any alerts. We'll see if we get a request for this.

File: cyences_app_for_splunk/default/data/ui/views/cs_windows_reports.xml (hunk 1 above)
Comment (VatsalJagani) [reply cyences_app_for_splunk/default/data/ui/views/cs_windows_reports.xml:48]: Removed duplicated search from line no. 7.

**Review by babhalodia — APPROVED** [review]

**Pull request merged**